parm/stage/atmosphere_warm_RT.yaml.j2

```
@@ -2,7 +2,7 @@ atmosphere_warm_RT:
     mkdir:
         - "{{ COMOUT_ATMOS_RESTART_PREV_MEM }}"
         - "{{ COMOUT_ATMOS_ANALYSIS_MEM }}"
-    copy:
+    link:
```

```suggestion
    link_req:
```
Use `link_req` to ensure that it checks the file is in place.